edpm_ansible/roles/test_deps/vars/centos-8.yml

```diff
@@ -21,6 +21,6 @@ test_deps_yaml_packages:
 test_deps_selinux_packages:
   - python3-libselinux
 test_deps_edpm_packages:
-  - python3-edpmclient
+  - python3-tripleoclient
   - python3-openstacksdk
-test_deps_edpm_repos: current-edpm
+test_deps_tripleo_repos: current-tripleo
```

Comment:
this var is not used so can be removed, I think

Reply:
same as above.